src/components/UserTabs/UserCard/UserCard.jsx

```diff
@@ -40,30 +42,31 @@ const UserCard = ({ user: { id, name, avatar, recipes } }) => {
             alt={name + "'s photo"}
             defaultImage={defaultAvatar}
           />
-          <div className={css.selected_user}></div>
+          {/* <div className={css.selected_user}></div> */}
```

Comment:
remove comment

Reply:
Left until final decision about new element is adopted